modules/homa-lite/src/lib.rs

```diff
@@ -894,7 +939,8 @@ pub mod module {
 				);
 				let actual_staking_amount = Self::convert_liquid_to_staking(actual_liquid_amount)?;
 
-				TotalStakingCurrency::<T>::mutate(|x| *x = x.saturating_sub(actual_staking_amount));
+				total_staking_currency = total_staking_currency.saturating_sub(actual_staking_amount);
+				Self::update_total_staking_currency_storage(|_n| Ok(total_staking_currency))?;
```

Comment:
why not do the original way?
```suggestion
				Self::update_total_staking_currency_storage(|n| Ok(n.saturating_sub(total_staking_currency)))?;
```

Reply:
Yeah i think at some point `total_staking_currency` was used somewhere else. But his is redundant now. so updated to do the modification in the closure.